src/TimeSeries/TimeSeries.xml

```diff
@@ -0,0 +1,121 @@
+                </property>
+                <property key="seriesKey" type="string">
+                    <caption>Name</caption>
+                    <category>Data source</category>
+                    <description>Used in the legend to interact with the chart</description>
+                </property>
+                <property key="entityConstraint" type="entityConstraint" entityProperty="seriesEntity" required="false">
+                    <caption>Constraint</caption>
+                    <category>Source - XPath</category>
+                    <description>Constraint on the data from the data entity</description>
+                </property>
+                <property key="dataSourceMicroflow" type="microflow" required="false">
+                    <caption>Data source microflow </caption>
+                    <category>Source - microflow</category>
+                    <description>Returns a series' coordinates</description>
+                    <returnType type="Object" entityProperty="seriesEntity" isList="true"/>
+                </property>
+                <property key="seriesColor" type="string" required="false">
+                    <caption>Series color</caption>
+                    <category>Appearance</category>
+                    <description>Color of the line in HTML colors; color name, RGB or HEX</description>
+                </property>
+                <property key="seriesFill" type="boolean" defaultValue="false">
+                    <caption>Fill area</caption>
+                    <category>Appearance</category>
+                    <description>Should series area be filled</description>
+                </property>
+            </properties>
+        </property>
+        <property key="width" type="integer" defaultValue="920">
+            <caption>Chart width</caption>
+            <category>Appearance</category>
+            <description>Width of the chart</description>
+        </property>
+        <property key="widthUnit" type="enumeration" defaultValue="auto">
+            <caption>Width unit</caption>
+            <category>Appearance</category>
+            <description/>
+            <enumerationValues>
+                <enumerationValue key="auto">Auto</enumerationValue>
+                <enumerationValue key="pixels">Pixels</enumerationValue>
+            </enumerationValues> 
+        </property>
+        <property key="height" type="integer" defaultValue="500">
+            <caption>Chart height</caption>
+            <category>Appearance</category>
+            <description>Height of the chart</description>
+        </property>
+        <property key="heightUnit" type="enumeration" defaultValue="auto">
+            <caption>Height unit</caption>
+            <category>Appearance</category>
+            <description/>
+            <enumerationValues>
+                <enumerationValue key="auto">Auto</enumerationValue>
+                <enumerationValue key="pixels">Pixels</enumerationValue>
```

Comment:
When the enum value is "Auto", we use a dimension of 100% of the parent's height, and width.